.travis.yml

```diff
@@ -23,7 +23,7 @@ env:
 before_install:
   - brew update
   - brew outdated cmake || brew upgrade cmake
-  - brew install cppcheck doxygen ninja
```

Comment:
qt5 won't work that, it needs an export on CMAKE_PREFIX_PATH. Check according projects :)